Godeps/Godeps.json

```diff
@@ -7,7 +7,7 @@
 	"Deps": [
 		{
 			"ImportPath": "github.com/gogo/protobuf/proto",
-			"Rev": "ab6cea4a44ef42b748cd88d2d372047b75806e0c"
```

Comment:
I think so. @vieux had initially suggested to run `godep update github.com/gogo/protobuf/proto`.